lib/binding/kvdb_interface.c

```diff
@@ -698,6 +762,29 @@ hse_kvdb_storage_add(const char *kvdb_home, size_t paramc, const char *const *co
     return err;
 }
 
+const char *
+hse_kvs_name_get(struct hse_kvs *const handle)
+{
+    if (HSE_UNLIKELY(!handle))
+        return NULL;
+
+    return kvdb_kvs_name((struct kvdb_kvs *)handle);
+}
```

Comment:
I really do not like casting away constness, seems to me the API definition is in error..

Reply:
The const after the `*` is worthless in function declarations. Only useful in function definitions.

Reply:
In this case it is casting from _hse_kvs_ handle to _kvdb_kvs_ handle..